Makefile

```diff
@@ -50,6 +50,8 @@ PROTOC_RELEASE_BASE = https://github.com/protocolbuffers/protobuf/releases/downl
 GO = go
 GO_BIN := $(GOPATH)/bin
 GO_SRC := $(GOPATH)/src
+# Defines the absolute local directory of the open-match project
+OM_DIR := $(dir $(abspath $(MAKEFILE_LIST)))
```

Comment:
OM_DIR isn't very descriptive. Can you change to REPOSITORY_ROOT or something else?